selectrum.el

```diff
@@ -88,6 +75,11 @@ See `minibuffer-default-in-prompt-regexps', from which this is derived.")
   :prefix "selectrum-"
   :link '(url-link "https://github.com/raxod502/selectrum"))
 
+(defcustom selectrum-should-sort-p t
```

Comment:
Should this really be a defcustom? I think most of the variables which are supposed to be locally tuned in the minibuffer-with-setup-hook should be defvar-local. The same applies to selectrum--move-default-candidate for example. Is this global for legacy reasons? Then I would probably keep it as a defvar, add a comment that this variable is to be deprecated. And later on deprecate the variable, replacing it by a local variant. At the same time one could get rid of the `-p` suffix which goes against the elisp style guide.

Reply:
I think it should be a defcustom the variables are able to be set locally but first and foremost are meant as user options. The name is problematic but I would prefer to avoid inconveniences for this reasons

Reply:
Okay, I am only aksing for this reason - if it is really intended as a user option, fine! I still have my doubts that it is useful as a global user option. Same applies to selectrum-move-default-candidate, which you will probably make public at some point (#333).

Reply:
I know about users which want to be able to disable sort by default that is why I think it should be an user option. For `selectrum-move-default-candidate,` I don't think it should be an option as its meant for session configuration.